package/boot/uboot-layerscape/Makefile

```diff
@@ -6,16 +6,15 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=uboot-layerscape
-PKG_SOURCE_DATE:=2017-12-07
+PKG_VERSION:=lsdk-1806
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/qoriq-open-source/u-boot.git
-PKG_SOURCE_VERSION:=9f7df1b406ff11409021cd2112beedd6b57bb600
-PKG_MIRROR_HASH:=b3756f814b731af2d03b0582ece90b2de564955b778f341fbfc34fa9bd849819
+PKG_SOURCE_URL:=https://source.codeaurora.org/external/qoriq/qoriq-components/u-boot
```

Comment:
How long will your repositories be available at https://source.codeaurora.org ? 
I though the merger between NXP and Qualcomm stopped?

Reply:
Yes, you're right.
But as I know we will still use CAF for the futhure layerscape release.